ucm2/ucm.conf

```diff
@@ -40,11 +41,40 @@ If.driver {
 		#
 		# The probed path when hw-card is found:
 		#
-		#   ucm2/${KernelModule}/${KernelModule}.conf
-		#   ucm2/${CardDriver}/${CardLongName}.conf
-		#   ucm2/${CardDriver}/${CardDriver}.conf
+		#   ucm2/conf.d/[${CardDriver}|${KernelDriver}]/${CardLongName}.conf
+		#   ucm2/conf.d/[${CardDriver}|${KernelDriver}]/[${CardDriver}|${KernelDriver}].conf
```

Comment:
Could you please point, how is the `${CardDriver}` handled? In your code I see only `ucm2/conf.d/${KernelDriver}/${CardLongName}.conf` and `ucm2/conf.d/${KernelDriver}/${KernelDriver}.conf`